caiman_napari/main_offline_gui.py

```diff
@@ -103,11 +108,21 @@ def run_item(self):
 
         ix = self.dataframe[self.dataframe['uuid'] == uuid].index[0]
 
-        self._run_index(ix)
+        # Run every item starting from the selected item
+        for i in (ix, self.ui.listWidgetItems.count()):
+            self.set_list_widget_item_color(i, 'orange')
+            try:
+                self._run_index(i)
```

Comment:
No this will launch every batch item immediately. When a batch item finishes we need to use a callback function that ends up running the next function.

Reply:
I can show you how it's done in Mesmerize